src/stores/connectors/secrets/get.go

```diff
@@ -11,6 +12,14 @@ import (
 func (c Connector) Get(ctx context.Context, id, version string) (*entities.Secret, error) {
 	logger := c.logger.With("id", id, "version", version)
 
+	if version == "" {
```

Comment:
This should be done after the authorization check

Reply:
agree